packages/core/src/__integrationtests__/utils.ts

```diff
@@ -67,3 +90,111 @@ export const IsOfficialLicenseAuthority = CType.fromSchema({
   },
   type: 'object',
 })
+
+export async function initialTransfer(
+  faucet: KeyringPair,
+  addresses: Array<string>
+): Promise<void> {
+  await Promise.all(
+    addresses.map((address) =>
+      Balance.makeTransfer(address, ENDOWMENT).then((tx) =>
+        BlockchainUtils.signAndSubmitTx(tx, faucet, {
+          resolveOn: BlockchainUtils.IS_FINALIZED,
+          reSign: true,
+        }).catch((e) => console.log(e))
+      )
+    )
+  )
+}
+
+export async function getDidDeposit(didIdentifier: string): Promise<BN> {
+  const { api } = await BlockchainApiConnection.getConnectionOrConnect()
+  const result = await api.query.did.did<Option<any>>(didIdentifier)
+  DecoderUtils.assertCodecIsType(result, ['Option<DidDidDetails>'])
+  return result.isSome ? result.unwrap().deposit.amount.toBn() : new BN(0)
+}
+
+export async function getAttestationDeposit(claimHash: string): Promise<BN> {
```

Comment:
Same as above. Feel free to resolve if not applicable.

Reply:
Again updated. I have had to export the `rawQuery` from the attestation to get access to the deposit.